spec/Observable-spec.ts

```diff
@@ -193,6 +193,80 @@ describe('Observable', () => {
       source.subscribe();
     });
 
+    it('should work when subscribe is called with no arguments and other subscription with no arguments completes', (done: MochaDone) => {
+      let nextCalled = false;
+      const source = new Observable((subscriber: Rx.Subscriber<string>) => {
+        const id = setTimeout(() => {
+          subscriber.next('foo');
+          subscriber.complete();
+          nextCalled = true;
+        }, 1);
+        return () => {
+          clearTimeout(id);
+        };
+      });
+
+      source.subscribe();
+
+      Observable.empty().subscribe();
+
+      setTimeout(() => {
+        let err;
+        let errHappened = false;
+        try {
```

Comment:
This try-catching seems unnecessary. Maybe I'm misunderstanding the purpose. Can you clairify?